engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -780,34 +766,19 @@ public RenderingStage getCurrentRenderStage() {
         return currentRenderingStage;
     }
 
+    /**
+     * Forces a recompilation of all shaders. This command, backed by Gestalt's monitoring feature,
+     * allows developers to hot-swap shaders for easy development.
+     */
```

Comment:
How about we get in the habit of providing examples of the command?
I.e. in this case we can add in the comment:

`* To run the command simply type "recompileShaders" + Enter in the consolle. `